Documentation/building/cross-building.md

```diff
@@ -1,7 +1,23 @@
-Cross Compilation for ARM
-=========================
+Cross Compilation for ARM on Linux
```

Comment:
The doc is really for targeting arm so I would favor keeping it until the document is updated to cross compile for more architectures.